### PR TITLE
Move Port and Module classes to general libbess-python.

### PR DESCRIPTION
**Pull request by justinemarie**
I have some thoughts about this for @sangjinhan and @melvinw that I'll bring up in person.

### CODE REVIEW

File: libbess-python/sugar.py [hunk 1]
```diff
@@ -374,5 +374,6 @@ def _run_tests():
     _test(env_suite)
     _test(mod_suite)
 
+
```
Comment (justinemarie) [inline libbess-python/sugar.py]: This is auto-added by pep8.

File: bessctl/sugar.py [hunk 2]
```diff
@@ -374,5 +374,6 @@ def _run_tests():
     _test(env_suite)
     _test(mod_suite)
 
+
```
Comment (justinemarie) [inline bessctl/sugar.py]: This was auto-added by pep8.

**Comment by justinemarie** [comment]
lol thx for the rename

**Comment by justinemarie** [comment]
Rebased to pull in the RunSplit revert.

**Comment by sangjinhan** [comment]
tumblr_m7nfbim9ml1qz4dkmo1_1280

**Pull request merged**